docs/REFERENCE.md

```diff
@@ -713,17 +713,18 @@ chezmoi's configuration file.
 
 chezmoi provides the following automatically populated variables:
 
-| Variable                | Value                                                                                                                  |
-| ----------------------- | ---------------------------------------------------------------------------------------------------------------------- |
-| `.chezmoi.arch`         | Architecture, e.g. `amd64`, `arm`, etc. as returned by [runtime.GOARCH](https://godoc.org/runtime#pkg-constants).      |
-| `.chezmoi.fullHostname` | The full hostname of the machine chezmoi is running on.                                                                |
-| `.chezmoi.group`        | The group of the user running chezmoi.                                                                                 |
-| `.chezmoi.homedir`      | The home directory of the user running chezmoi.                                                                        |
-| `.chezmoi.hostname`     | The hostname of the machine chezmoi is running on, up to the first `.`.                                                |
-| `.chezmoi.os`           | Operating system, e.g. `darwin`, `linux`, etc. as returned by [runtime.GOOS](https://godoc.org/runtime#pkg-constants). |
-| `.chezmoi.osRelease`    | The information from `/etc/os-release`, Linux only, run `chezmoi data` to see its output.                              |
-| `.chezmoi.sourceDir`    | The source directory.                                                                                                  |
-| `.chezmoi.username`     | The username of the user running chezmoi.                                                                              |
+| Variable                | Value                                                                                                                         |
+| ----------------------- | ----------------------------------------------------------------------------------------------------------------------------- |
+| `.chezmoi.arch`         | Architecture, e.g. `amd64`, `arm`, etc. as returned by [runtime.GOARCH](https://godoc.org/runtime#pkg-constants).             |
+| `.chezmoi.fullHostname` | The full hostname of the machine chezmoi is running on.                                                                       |
+| `.chezmoi.group`        | The group of the user running chezmoi.                                                                                        |
+| `.chezmoi.homedir`      | The home directory of the user running chezmoi.                                                                               |
+| `.chezmoi.hostname`     | The hostname of the machine chezmoi is running on, up to the first `.`.                                                       |
+| `.chezmoi.kernel`       | Contains information from `/proc/sys/kernel`. Linux only, useful for detecting specific kernels (I.e. microsfot's WSL kernel) |
```

Comment:
Please fix the typos and capitalization and add a closing full stop: `(i.e. Microsoft's WSL kernel).`